src/js/components/Number/doc.js

```diff
@@ -0,0 +1,112 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { colorPropType } from '../../utils/prop-types';
+import { getAvailableAtBadge } from '../../utils/mixins';
+
+export const doc = Number => {
+  const DocumentedNumber = describe(Number)
+    .availableAt(getAvailableAtBadge('Number', 'Visualizations'))
+    .description('Is a number value with optional units')
+    .usage(
+      `import { Number } from 'grommet';
+<Number value={27} units="GB" />`,
+    );
+
+  DocumentedNumber.propTypes = {
+    color: colorPropType.description(
+      'A color identifier to use for the text color.',
+    ),
+    size: PropTypes.oneOfType([
+      PropTypes.oneOf(['small', 'medium', 'large', 'xlarge']),
+      PropTypes.string,
+    ])
+      .description('The size of the text.')
```

Comment:
I think we want to be a little more accurate and say it is for the 'value text' instead of just 'text'. Or in cases, it affects both value-text and unit-text, we should say something like 'the Number text'